src/content/docs/agents/java-agent/configuration/java-agent-configuration-config-file.mdx

```diff
@@ -2626,6 +2626,46 @@ These options are set in the `error_collector` stanza and unless noted otherwise
     ```
   </Collapser>
 
+  <Collapser
+    id="ec-ignoreErrorPriority"
+    title="ignoreErrorPriority"
+  >
+    <table>
+      <tbody>
+        <tr>
+          <th>
+            Type
+          </th>
+
+          <td>
+            Boolean
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            Default
+          </th>
+
+          <td>
+            false
+          </td>
+        </tr>
+      </tbody>
+    </table>
+
+    With default ignoreErrorPriority as false, the agent reports the last error noticed if noticeError() is called multiple times in a transaction. If set to true, the agent reports the first error noticed if noticeError() is called multiple times in a transaction.
+
+    This setting is dynamic, so running agents will notice changes to `newrelic.yml` without a JVM restart. <--- need to verify for this one
```

Comment:
It looks like the `<---` is breaking our build check @XiXiaPdx. You'll definitely want to remove that once you're ready for this to go out.